config/settings/local.py

```diff
@@ -21,9 +21,17 @@
     "default": {
         "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
         "LOCATION": "",
+    },
```

Comment:
We should add these tos settings in `settings/base.py` and use `redis` as a cache for it. Then, we don't need to override the settings `local` and `production` as we have redis in both the places.